src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/ExactTypeBindingProvider.cs

```diff
@@ -96,7 +102,14 @@ private class ExactBinding : BindingBase<TAttribute>
 
                 IValueProvider vp = new ConstantValueProvider(obj, typeof(TUserType), invokeString);
                 return vp;
-            }            
+            }
+
+            protected override Task<IValueProvider> BuildAsync(TAttribute attrResolved, BindingContext context)
+            {
+                _preBindHook?.Invoke(attrResolved, context);
```

Comment:
Not sure I understand - BindingContext is only available at invoke time, whereas up to this point all of the other binding logic (other than converters) was index time.